src/extra/components/xgm.py

```diff
@@ -319,6 +325,8 @@ def pulse_energy(self, sase=None):
                 `data.intensitySa1TD` with `sase=1`. This setting overrides the
                 `default_sase` argument to the
                 [constructor][extra.components.XGM].
+            series (bool): Wherther to return a 2D [DataArray][xarray.DataArray]
```

Comment:
👍

Reply:
I disagree for two reasons:
- Personally I dislike it when some vague term is used that implies completely different behaviour, like returning a `Series` instead of a `DataArray`.
- `flatten` is ambiguous because it could also mean returning a 1D `DataArray` with a `pd.MultiIndex`, rather than a `Series`.

An alternative would be `flatten_as_series`, though that's a bit long for my taste.